docs/AA--Javascript-in-EE/components/form/select/editor-event-select.md

```diff
@@ -0,0 +1,64 @@
+
+
+# `<EditorEventSelect />`
+
+**EditorEventSelect** applies WordPress admin formatting to the [EventSelect](event-select.md) component by wrapping it with the Gutenberg [BaseControl](https://github.com/WordPress/gutenberg/tree/master/components/base-control) component. This wil override the label applied by the **EventSelect** component in favour of the one applied by the **BaseControl** component. Help text can also be specified which will be displayed after the **EventSelect** component.
+
+The **EditorEventSelect** component accepts **all** of the properties that the the **EventSelect** component does as well as the following:
+
+```jsx
+<EditorEventSelect
+    id="datetime-select-1" 
+    className="datetime-select"
+    help={ 'This text will be displayed after the select input to aid the user in understanding its purpose or effect.' }
+/>
+```
+
+
+## Props
+
+The component accepts the following props:
+
+### id
+
+The id of the element to which labels and help text are being generated. That element should be passed as a child.
+
+- Type: `String`
+- Required: Yes
+
```

Comment:
This prop is no longer accepted/handled.

Reply:
To save having to add more comments in the other affected docs just note that this applies to any other `Editor*Select` Component docs.